src/web/components/ForgotPasswordConfirmForm.jsx

```diff
@@ -29,7 +29,7 @@ export default function ForgotPasswordConfirmForm(props: Object): React.Element
                 placeholder="e.g. 12345678"
                 value={confirmationCode}
                 onChange={onChange('confirmationCode')}
-                inputProps={{autocomplete: "off"}}
+                inputProps={{autoComplete: "off"}}
```

Comment:
Maybe revert these `autoComplete`s back to `autocomplete`, the spec defines it as lowercase: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete Browsers may be able to cope with mixed case html attribute names so it may still work, but probably not worth changing.